modules/gridNMBidAdapter.md

```diff
@@ -26,11 +28,7 @@ Grid bid adapter supports Banner and Video (instream and outstream).
                        params: {
                            source: 'jwp',
                            secid: '11',
-                           pubid: '22',
-                           video: {
-                               mimes: ['video/mp4', 'video/x-ms-wmv'],
-                               protocols: [1,2,3,4,5,6]
-                           }
```

Comment:
Probably best to still include an example of being able to pass video property via your params since your adapter still grabs from both sources.